packages/rocketchat-lib/server/methods/sendInvitationEmail.js

```diff
@@ -9,8 +9,9 @@ Meteor.startup(() => {
 	});
 });
 
+/* eslint no-unused-vars: ["error", { "args": "none" }] */
 Meteor.methods({
-	sendInvitationEmail(emails) {
+	sendInvitationEmail(emails, language) {
```

Comment:
is the new "language" parameter being used here?  Perhaps I don't understand how this works, but I'm not seeing it...

Reply:
No this is not being used in this method, it is being used in SMS invite, and is just available to Email invite method, the method populates the body on startup, so should we change the method altogether to support this?

Reply:
We do want the email message to be in the senders default language........  so yes, if that is the best way to do it, we should adjust the method to use it I would think.

Reply:
Please review the SMS Invite part of the PR, if it works properly, how about we go ahead and merge this and corresponding kotlin sdk, and android PRs. Then I can create a seperate PR for handling the Email case

Reply:
OK, I'll test

Reply:
@kb0304 I still cannot get this to work.......  any ideas?